content/flux/v0.x/stdlib/experimental/aggregate/rate.md

```diff
@@ -5,6 +5,7 @@ description: >
 aliases:
   - /influxdb/v2.0/reference/flux/stdlib/experimental/aggregate/rate/
   - /influxdb/cloud/reference/flux/stdlib/experimental/aggregate/rate/
+  - /flux/v0.x/stdlib/experimental/to/aggregate/rate/
```

Comment:
@sanderson, do you recall if this was a legit structure in the past?
 
Or perhaps a random typo in the `aggregate.rate()` link (--appears in v2.0 and 2.1 in the [rate.md](https://github.com/influxdata/docs-v2/blob/master/content/influxdb/v2.1/query-data/flux/rate.md) file)?